UPGRADE-1.10.md

```diff
@@ -9,3 +9,5 @@
    If you are using they on your custom entity `api_resource` configuration or serialization groups, you should check if one of these changes may affect on your app. If yes, change all occurs by this pattern:
 
 - `product_review:update` changed to: `admin:product_review:update` and `shop:product_review:update`
+
+1. We've removed `Sylius\Bundle\ApiBundle\ApiPlatform\Bridge\Doctrine\ORM\SubresourceDataProvider` is not longer needed because `ApiPlatform\Core\Bridge\Doctrine\Orm\SubresourceDataProvider` is the same file.
```

Comment:
```suggestion
1. We've removed `Sylius\Bundle\ApiBundle\ApiPlatform\Bridge\Doctrine\ORM\SubresourceDataProvider`. It's no longer needed because `ApiPlatform\Core\Bridge\Doctrine\Orm\SubresourceDataProvider` is the same file.
```